Guesser/FilterTypeGuesser.php

```diff
@@ -62,7 +66,7 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
             $options['operator_type'] = 'sonata_type_boolean';
             $options['operator_options'] = array();
 
-            $options['field_type'] = 'document';
+            $options['field_type'] = DocumentType::class;
```

Comment:
`document` never existed afaik. I guess `phpcr_document` was meant to be used (so in fact, nobody ever used this filter)